### DISCUSSION TITLE
how to access your website?

### DISCUSSION THREAD

**Discussion by Wjxfi** [SuhasDissa/Food-E-App#50] [General]
give me a link to it, please. And put the link in a very prominent place so that it is useful

**Comment by SuhasDissa** [comment]
[Railway](https://railway.app) discontinued their free tier. So my website stopped working since then.
I will find a different hosting service and infrom you soon.

**Comment by SuhasDissa** [comment]
The website is still under development, so editing features aren't available yet. But I'm working on it!

I wanted to let you know that I rewrote the whole website from scratch. The original website was built with PHP, and it was really slow and difficult to host. So I'm currently rewriting the whole website in SvelteKit, which is a much faster and lightweight framework.

I'll keep you updated on my progress. Thanks for caring.

Here are the links to my repos, Kindly post future issues there.
[Food-E-Web](https://github.com/SuhasDissa/Food-E-Web)
[Food-E-Web (Depricated)](https://github.com/SuhasDissa/Food-E-Web-Depricated)

**Comment by SuhasDissa** [comment]
Hi there!

I'm excited to share that I've hosted Food-E website as a static site. This means it's now lightning-fast ⚡  and reliable, and it didn't cost a penny!

The website URL is now in the  Food-E-Web repository details, so you can check it out.

Thank You,
